public/stylesheets/style.css

```diff
@@ -6,7 +6,9 @@ body {
   margin: 2em;
 }
 
-
+.poster_list {
```

Comment:
Name of this class needs to follow BEM methodology, like I mentioned in the last comment. Using random names is generally a recipe for CSS thats impossible to follow along and maintain. Hence the pain. Sorry, let's fix this.